app/code/Tasks/FeedbackManager/Controller/Manage/Delete.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Tasks\FeedbackManager\Controller\Manage;
+
+use Magento\Customer\Controller\AbstractAccount;
+use Magento\Framework\App\Action\Context;
+use Magento\Customer\Model\Session;
+
+class Delete extends \Magento\Framework\App\Action\Action
+{
+    protected $feedbackFactory;
+    protected $customerSession;
+    protected $messageManager;
+    protected $jsonData;
+
+    public function __construct(
+        Context $context,
+        \Tasks\FeedbackManager\Model\FeedbackFactory $feedbackFactory,
+        Session $customerSession,
+        \Magento\Framework\Json\Helper\Data $jsonData
+    ) {
+        $this->feedbackFactory = $feedbackFactory;
+        $this->customerSession = $customerSession;
+        $this->jsonData = $jsonData;
+        parent::__construct($context);
+    }
+
+    public function execute()
+    {
+        $feedbackId = $this->getRequest()->getParam('id');
+        $customerId = $this->customerSession->getCustomerId();
+        $isAuthorised = $this->feedbackFactory->create()
+                                    ->getCollection()
+                                    ->addFieldToFilter('user_id', $customerId)
+                                    ->addFieldToFilter('entity_id', $feedbackId)
+                                    ->getSize();
+        if (!$isAuthorised) {
+            $msg=__('You are not authorised to delete this feedback.');
+            $success=0;
+        } else {
+            $model = $this->feedbackFactory->create()->load($feedbackId);
+            $model->delete();
+            $msg=__('You have successfully deleted the feedback.');
+            $success=1;
+        }     
+        $this->getResponse()->setHeader('Content-type', 'application/javascript');
+        $this->getResponse()->setBody(
+            $this->jsonData->jsonEncode(
```

Comment:
@gayathri-aravind Use serializer interface instead of this one

Reply:
Used \Magento\Framework\Serialize\Serializer\Json interface. Commit: 882571dc9e3988e3e1f109a2d06fe316213a2971